include/oneapi/dpl/pstl/utils.h

```diff
@@ -765,6 +765,24 @@ struct __is_iterator_type<_T, std::void_t<typename std::iterator_traits<_T>::dif
 template <typename _T>
 static constexpr bool __is_iterator_type_v = __is_iterator_type<_T>::value;
 
+template <typename _Tp>
+union __lazy_ctor_storage
+{
+    using __value_type = _Tp;
+    _Tp __v;
+    __lazy_ctor_storage() {}
+    void
+    __setup(const _Tp& init)
```

Comment:
Since we have this function now, I think we can replace the placement news in the `transform_reduce` struct in `unseq_backend_sycl.h` to call this function for consistency.

Reply:
Oh yes, this was part of the original changes in the transform_scan branch,  I didn't see those were missing here.

Reply:
Good catch. I made those changes.